### PR TITLE
adding sonaqube dashboard  into our qa dashboard 

### PR DESCRIPTION
**Pull request by sherrif10**
cc @kdaud  because it was merged perhaps without reviews feel free to add more comments in this  new pr  thanks.

### CODE REVIEW

**Comment by sherrif10** [comment]
Appologies , @kdaud ,  feel at peace to do the needful upon this pr.

**Comment by sherrif10** [comment]
@kdaud  Just incase you find this sufficient, we get this moved home

**Comment by kdaud** [comment]
should we put openmrs sonaqube status budge under `Install & Upgrade Tests?` or Under `Platform(core)`?

**Comment by sherrif10** [comment]
Added it into platform.

File: README.md [hunk 1]
```diff
@@ -5,18 +5,19 @@
 ___
 ## Platform (core)
 
-[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2)
+[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2) | [sonaqube dashboard](https://ci.openmrs.org/browse/SON-OPENMRSCOREMASTER)
+
```
Comment (kdaud) [inline README.md:9]: why not capitalize the word `sonaqube dashboard` to suit the existing naming convention for the Plaform(core) budges?

**Comment by sherrif10** [comment]
@kdaud do you time look through this probably require approval or recommendation

**Comment by sherrif10** [comment]
Done.

File: README.md [hunk 2]
```diff
@@ -5,18 +5,19 @@
 ___
 ## Platform (core)
 
-[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2)
+[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2) | [SONAQUBE-DASHBOARD](https://ci.openmrs.org/browse/SON-OPENMRSCOREMASTER)
```
Comment (kdaud) [inline README.md:8]: do we need  a dash(`-`) between the two words?
```suggestion
[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2) | [SONAQUBE DASHBOARD](https://ci.openmrs.org/browse/SON-OPENMRSCOREMASTER)
```

**Comment by kdaud** [comment]
could be missing some 
Screenshot from 2021-11-18 17-17-02
syntax in the markdown code that is causing the Platform(core) Dashboard to look like this?

**Comment by sherrif10** [comment]
Resolved.

**Comment by kdaud** [comment]
> Resolved.

sure? 

Screenshot from 2021-11-18 17-52-58

**Comment by sherrif10** [comment]
is there anything missing from that

**Comment by kdaud** [comment]
> Resolved.
> is there anything missing from that

take a look and confirm!

Screenshot from 2021-11-18 17-52-58

**Comment by sherrif10** [comment]
is there anything missing again, Have you seen my comment above.

**Comment by kdaud** [comment]
> These wording wont be on the same line as before reason is the word sona qube have more words they can be put in

Are you sure? won't you just be missing some markdown syntax that can do the job for you?

Borrow a leaf on the existing syntax for Platform(core) and do the needful!

**Comment by kdaud** [comment]
@sherrif10 do you mean if the wording was short, it would be as desired?

**Comment by sherrif10** [comment]
Done.

**Review by kdaud — APPROVED** [review]

**Pull request merged**